### PR TITLE
Fix how add_npm_package determines if an update is needed

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: add_npm_package.sh [hunk 1]
```diff
@@ -138,7 +139,7 @@ if [[ -z $STRATEGY ]] ; then
   echo "Found $DEPENDENCIES dependencies - using $STRATEGY strategy"
 fi
 
-if [ -f "$PACKAGE_DIR"/*.spec ]; then
```
Comment (ekohl) [inline add_npm_package.sh:141]: Context: I think this predates from when we properly aligned all spec files to match their directory name. These days we can rely on that.

File: add_npm_package.sh [hunk 2]
```diff
@@ -18,6 +18,7 @@ else
   PACKAGE_NAME=nodejs-${PACKAGE_MODULE}
 fi
 PACKAGE_DIR=packages/$BASE_DIR/$PACKAGE_NAME
+SPEC_FILE="${PACKAGE_NAME}.spec"
```
Comment (ekohl) [inline add_npm_package.sh:21]: Should we make this an absolute path?

**Review by ekohl — CHANGES_REQUESTED** [review]
Can you also use the new variable on line 150?

**Comment by ehelms** [comment]
> Can you also use the new variable on line 150?

Done

**Pull request merged**